simulator_files/slurm_sim_ws/bld_opt/makeconfigurelog.txt
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking whether to include rpath in build... yes
checking for style of include used by make... GNU
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking dependency style of gcc... gcc3
checking for mysql_config... no
checking for mariadb_config... /usr/bin//mariadb_config
checking whether BG is explicitly disabled... no
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking for gcc... (cached) gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to accept ISO C89... (cached) none needed
checking dependency style of gcc... (cached) gcc3
checking whether we are using the GNU C++ compiler... (cached) yes
checking whether g++ accepts -g... (cached) yes
checking dependency style of g++... (cached) gcc3
checking whether make sets $(MAKE)... (cached) yes
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for pkg-config... no
checking for sleep... /bin/sleep
checking for su... /bin/su
checking for library containing socket... none required
checking for library containing gethostbyname... none required
checking for library containing hstrerror... none required
checking for library containing kstat_open... no
checking mcheck.h usability... yes
checking mcheck.h presence... yes
checking for mcheck.h... yes
checking values.h usability... yes
checking values.h presence... yes
checking for values.h... yes
checking socket.h usability... no
checking socket.h presence... no
checking for socket.h... no
checking sys/socket.h usability... yes
checking sys/socket.h presence... yes
checking for sys/socket.h... yes
checking stdbool.h usability... yes
checking stdbool.h presence... yes
checking for stdbool.h... yes
checking sys/ipc.h usability... yes
checking sys/ipc.h presence... yes
checking for sys/ipc.h... yes
checking sys/shm.h usability... yes
checking sys/shm.h presence... yes
checking for sys/shm.h... yes
checking sys/sem.h usability... yes
checking sys/sem.h presence... yes
checking for sys/sem.h... yes
checking errno.h usability... yes
checking errno.h presence... yes
checking for errno.h... yes
checking for stdlib.h... (cached) yes
checking dirent.h usability... yes
checking dirent.h presence... yes
checking for dirent.h... yes
checking pthread.h usability... yes
checking pthread.h presence... yes
checking for pthread.h... yes
checking sys/prctl.h usability... yes
checking sys/prctl.h presence... yes
checking for sys/prctl.h... yes
checking sysint.h usability... no
checking sysint.h presence... no
checking for sysint.h... no
checking for inttypes.h... (cached) yes
checking termcap.h usability... no
checking termcap.h presence... no
checking for termcap.h... no
checking netdb.h usability... yes
checking netdb.h presence... yes
checking for netdb.h... yes
checking for sys/socket.h... (cached) yes
checking sys/systemcfg.h usability... no
checking sys/systemcfg.h presence... no
checking for sys/systemcfg.h... no
checking ncurses.h usability... no
checking ncurses.h presence... no
checking for ncurses.h... no
checking curses.h usability... no
checking curses.h presence... no
checking for curses.h... no
checking sys/dr.h usability... no
checking sys/dr.h presence... no
checking for sys/dr.h... no
checking sys/vfs.h usability... yes
checking sys/vfs.h presence... yes
checking for sys/vfs.h... yes
checking pam/pam_appl.h usability... no
checking pam/pam_appl.h presence... no
checking for pam/pam_appl.h... no
checking security/pam_appl.h usability... no
checking security/pam_appl.h presence... no
checking for security/pam_appl.h... no
checking sys/sysctl.h usability... yes
checking sys/sysctl.h presence... yes
checking for sys/sysctl.h... yes
checking pty.h usability... yes
checking pty.h presence... yes
checking for pty.h... yes
checking utmp.h usability... yes
checking utmp.h presence... yes
checking for utmp.h... yes
checking sys/syslog.h usability... yes
checking sys/syslog.h presence... yes
checking for sys/syslog.h... yes
checking linux/sched.h usability... yes
checking linux/sched.h presence... yes
checking for linux/sched.h... yes
checking kstat.h usability... no
checking kstat.h presence... no
checking for kstat.h... no
checking paths.h usability... yes
checking paths.h presence... yes
checking for paths.h... yes
checking limits.h usability... yes
checking limits.h presence... yes
checking for limits.h... yes
checking sys/statfs.h usability... yes
checking sys/statfs.h presence... yes
checking for sys/statfs.h... yes
checking sys/ptrace.h usability... yes
checking sys/ptrace.h presence... yes
checking for sys/ptrace.h... yes
checking float.h usability... yes
checking float.h presence... yes
checking for float.h... yes
checking sys/statvfs.h usability... yes
checking sys/statvfs.h presence... yes
checking for sys/statvfs.h... yes
checking for sys/wait.h that is POSIX.1 compatible... yes
checking whether time.h and sys/time.h may both be included... yes
checking for ANSI C header files... (cached) yes
checking library containing dlopen... checking for dlopen in -l... no
checking for dlopen in -ldl... yes
checking for program_invocation_name... yes
checking for ptrace64... no
checking for sched_setaffinity... yes
checking for numa_available in -lnuma... no
checking whether to enable PAM support... yes
checking for pam_get_user in -lpam... no
checking for misc_conv in -lpam_misc... no
checking whether to enable ISO 8601 time format support... yes
checking whether sbatch --get-user-env option should load .login... no
checking whether byte ordering is bigendian... no
checking for json installation... 
checking whether gcc needs -traditional... no
checking for __builtin_clzll... yes
checking for __builtin_ctzll... yes
checking for __builtin_popcountll... yes
checking for stdlib.h... (cached) yes
checking for GNU libc compatible malloc... yes
checking whether strerror_r is declared... yes
checking for strerror_r... yes
checking whether strerror_r returns char *... no
checking for fdatasync... yes
checking for hstrerror... yes
checking for strerror... yes
checking for mtrace... yes
checking for strndup... yes
checking for strlcpy... no
checking for strsignal... yes
checking for inet_aton... yes
checking for inet_ntop... yes
checking for inet_pton... yes
checking for setproctitle... no
checking for sysctlbyname... no
checking for cfmakeraw... yes
checking for setresuid... yes
checking for get_current_dir_name... yes
checking for faccessat... yes
checking for eaccess... yes
checking for statvfs... yes
checking for statfs... yes
checking whether hstrerror is declared... no
checking whether strsignal is declared... yes
checking whether sys_siglist is declared... no
checking for the pthreads library -lpthreads... no
checking whether pthreads work without any flags... no
checking whether pthreads work with -Kthread... no
checking whether pthreads work with -kthread... no
checking for the pthreads library -llthread... no
checking whether pthreads work with -pthread... yes
checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
checking if more special flags are required for pthreads... no
checking for PTHREAD_PRIO_INHERIT... yes
checking System dimensions... not set
checking for ofed installation... 
checking for h5cc... no
checking for h5pcc... no
checking for HDF5 type... neither
checking for HDF5 libraries... no
checking if zlib is wanted... yes
checking for inflateEnd in -lz... no
checking zlib.h usability... no
checking zlib.h presence... no
checking for zlib.h... no
checking if lz4 is installed... yes /usr/local /usr /opt/local /sw
checking for LZ4_compress_destSize in -llz4... no
checking lz4.h usability... no
checking lz4.h presence... no
checking for lz4.h... no
checking for hwloc installation... 
checking for pmix installation... 
checking for freeipmi installation... 
checking for sem_open in -lposix4... no
checking for rrdtool installation... 
checking for initscr in -lncurses... no
checking for initscr in -lcurses... no
checking for tgetent in -ltinfo... no
checking for CHECK... no
checking for pkg-config... no
checking for GLIB - version >= 2.7.1... no
*** A new enough version of pkg-config was not found.
*** See http://www.freedesktop.org/software/pkgconfig/
checking for pkg-config... no
checking for GTK+ - version >= 2.7.1... no
*** A new enough version of pkg-config was not found.
*** See http://pkgconfig.sourceforge.net
checking whether this is a Cray XT or XE system running on ALPS or ALPS simulator... no
checking for datawarp installation... 
checking for __progname... yes
checking whether or not developer options are enabled... no
checking whether debugging is enabled... no
checking whether memory leak debugging is enabled... no
checking whether to enable slurmd operation on a front-end... yes
checking whether debugger partial attach enabled... no
checking whether salloc should kill child processes at job termination... no
checking whether to disable salloc execution in the background... no
checking for slurmctld default port... 6817
checking for slurmd default port... 6818
checking for slurmdbd default port... 6819
checking for slurmctld default port count... 1
checking Checking NRT and PERMAPI header files... no
configure: Cannot support IBM NRT without nrt.h and permapi.h
checking whether to enable IBM NRT support... no
checking for job_attachpid in -ljob... no
checking for SGI job container support... no
checking for netloc installation... 
checking for lua... no
checking whether man2html is available... checking for man2html... no
checking for support of printf("%s", NULL)... yes
checking for whether to include readline suport... yes
checking for OpenSSL directory... /usr
checking whether to enable multiple-slurmd support... no
checking whether or not simulator mode is enabled... yes
checking for openpty in -lutil... yes
checking for blcr installation... 
checking for gawk... (cached) gawk
checking for curl-config... no
checking whether libcurl is usable... no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating auxdir/Makefile
config.status: creating contribs/Makefile
config.status: creating contribs/cray/Makefile
config.status: creating contribs/cray/csm/Makefile
config.status: creating contribs/lua/Makefile
config.status: creating contribs/mic/Makefile
config.status: creating contribs/pam/Makefile
config.status: creating contribs/pam_slurm_adopt/Makefile
config.status: creating contribs/perlapi/Makefile
config.status: creating contribs/perlapi/libslurm/Makefile
config.status: creating contribs/perlapi/libslurm/perl/Makefile.PL
config.status: creating contribs/perlapi/libslurmdb/Makefile
config.status: creating contribs/perlapi/libslurmdb/perl/Makefile.PL
config.status: creating contribs/seff/Makefile
config.status: creating contribs/torque/Makefile
config.status: creating contribs/openlava/Makefile
config.status: creating contribs/phpext/Makefile
config.status: creating contribs/phpext/slurm_php/config.m4
config.status: creating contribs/sgather/Makefile
config.status: creating contribs/sgi/Makefile
config.status: creating contribs/sjobexit/Makefile
config.status: creating contribs/pmi2/Makefile
config.status: creating doc/Makefile
config.status: creating doc/man/Makefile
config.status: creating doc/man/man1/Makefile

simulator_files/slurm_sim_ws/bld_opt/config.h
/* config.h.  Generated from config.h.in by configure.  */
/* config.h.in.  Generated from configure.ac by autoheader.  */

/* Define if building universal (internal helper macro) */
/* #undef AC_APPLE_UNIVERSAL_BUILD */

/* Define BLCR installation home */
#define BLCR_HOME ""

/* Define location of cpuset directory */
/* #undef CPUSET_DIR */

/* Define to 1 for debugger partial task attach support. */
#define DEBUGGER_PARTIAL_ATTACH 1

/* Define to 1 if using glib-2.32.0 or higher */
/* #undef GLIB_NEW_THREADS */

/* Define to 1 if licensed under terms of the GNU General Public License. */
#define GPL_LICENSED 1

/* Define to 1 if using gtk+-2.14.0 or higher */
/* #undef GTK2_USE_GET_FOCUS */

/* Define to 1 if using gtk+-2.10.0 or higher */
/* #undef GTK2_USE_RADIO_SET */

/* Define to 1 if using gtk+-2.12.0 or higher */
/* #undef GTK2_USE_TOOLTIP */

/* Make sure we get the 1.8 HDF5 API */
#define H5_NO_DEPRECATED_SYMBOLS 1

/* Define to 1 if 3-dimensional architecture */
/* #undef HAVE_3D */

/* Define to 1 if 4-dimensional architecture */
/* #undef HAVE_4D */

/* Define to 1 for Cray XT/XE systems using ALPS */
/* #undef HAVE_ALPS_CRAY */

/* Define to 1 for emulating a Cray XT/XE system using ALPS */
/* #undef HAVE_ALPS_CRAY_EMULATION */

/* Define to 1 if running against an ALPS emulation */
/* #undef HAVE_ALPS_EMULATION */

/* Define to 1 if emulating or running on Blue Gene system */
/* #undef HAVE_BG */

/* Define to 1 if emulating or running on Blue Gene/Q system */
/* #undef HAVE_BGQ */

/* Define to 1 if have Blue Gene files */
/* #undef HAVE_BG_FILES */

/* Define to 1 if using code where blocks have actions */
/* #undef HAVE_BG_GET_ACTION */

/* Define to 1 if using code with new iocheck */
/* #undef HAVE_BG_NEW_IO_CHECK */

/* Define to 1 if you have the `cfmakeraw' function. */
#define HAVE_CFMAKERAW 1

/* Define to 1 for systems with a Cray network */
/* #undef HAVE_CRAY_NETWORK */

/* Define to 1 if you have the <curses.h> header file. */
/* #undef HAVE_CURSES_H */

/* Define to 1 if DataWarp library found */
/* #undef HAVE_DATAWARP */

/* Define to 1 if you have the declaration of `hstrerror', and to 0 if you
   don't. */
#define HAVE_DECL_HSTRERROR 0

/* Define to 1 if you have the declaration of `strerror_r', and to 0 if you
   don't. */
#define HAVE_DECL_STRERROR_R 1

/* Define to 1 if you have the declaration of `strsignal', and to 0 if you
   don't. */
#define HAVE_DECL_STRSIGNAL 1

/* Define to 1 if you have the declaration of `sys_siglist', and to 0 if you
   don't. */
#define HAVE_DECL_SYS_SIGLIST 0

/* Define to 1 if you have the <dirent.h> header file. */
#define HAVE_DIRENT_H 1

/* Define to 1 if you have the <dlfcn.h> header file. */
#define HAVE_DLFCN_H 1

/* Define to 1 if you have the `eaccess' function. */
#define HAVE_EACCESS 1

/* Define to 1 if you have the <errno.h> header file. */
#define HAVE_ERRNO_H 1

/* Define to 1 if function EVP_MD_CTX_cleanup exists. */
/* #undef HAVE_EVP_MD_CTX_CLEANUP */

/* Define to 1 if you have the `faccessat' function. */
#define HAVE_FACCESSAT 1

/* Define to 1 if you have the `fdatasync' function. */
#define HAVE_FDATASYNC 1

/* Define to 1 if you have the <float.h> header file. */
#define HAVE_FLOAT_H 1

/* Define to 1 if freeipmi library found */
/* #undef HAVE_FREEIPMI */

/* Define to 1 if running slurmd on front-end only */
#define HAVE_FRONT_END 1

/* Define to 1 if you have the `get_current_dir_name' function. */
#define HAVE_GET_CURRENT_DIR_NAME 1

/* Defined if you have HDF5 support */
/* #undef HAVE_HDF5 */

/* Define to 1 if you have the `hstrerror' function. */
#define HAVE_HSTRERROR 1

/* Define to 1 if hwloc library found */
/* #undef HAVE_HWLOC */

/* Define to 1 if hwloc library supports PCI devices */
/* #undef HAVE_HWLOC_PCI */

/* Define to 1 if you have the `inet_aton' function. */
#define HAVE_INET_ATON 1

/* Define to 1 if you have the `inet_ntop' function. */
#define HAVE_INET_NTOP 1

/* Define to 1 if you have the `inet_pton' function. */
#define HAVE_INET_PTON 1

/* Define to 1 if you have the <inttypes.h> header file. */
#define HAVE_INTTYPES_H 1

/* Define if you are compiling with json. */
/* #undef HAVE_JSON */

/* Define if headers in include/json-c. */
/* #undef HAVE_JSON_C_INC */

/* Define if headers in include/json. */
/* #undef HAVE_JSON_INC */

/* Define to 1 if you have the <kstat.h> header file. */
/* #undef HAVE_KSTAT_H */

/* Define to 1 if you have a functional curl library. */
/* #undef HAVE_LIBCURL */

/* Define to 1 if you have `z' library (-lz) */
#define HAVE_LIBZ 1

/* Define to 1 if you have the <limits.h> header file. */
#define HAVE_LIMITS_H 1

/* Define to 1 if you have the <linux/sched.h> header file. */
#define HAVE_LINUX_SCHED_H 1

/* Compile with Lua 5.1 */
/* #undef HAVE_LUA_5_1 */

/* Compile with Lua 5.2 */
/* #undef HAVE_LUA_5_2 */

/* Define to 1 if you have 'lz4' library (-llz4) */
/* #undef HAVE_LZ4 */

/* Define to 1 if your system has a GNU libc compatible `malloc' function, and
   to 0 otherwise. */
#define HAVE_MALLOC 1

/* Define to 1 if you have the <mcheck.h> header file. */
#define HAVE_MCHECK_H 1

/* Define to 1 if you have the <memory.h> header file. */
#define HAVE_MEMORY_H 1

/* Define to 1 if you have the `mtrace' function. */
#define HAVE_MTRACE 1

/* Define to 1 if using MySQL libaries */
#define HAVE_MYSQL 1

/* Define to 1 for running on a Cray in native mode without ALPS */
/* #undef HAVE_NATIVE_CRAY */

/* Define to 1 if alpscomm functions new to CLE 5.2UP01 are defined */
/* #undef HAVE_NATIVE_CRAY_GA */

/* Define to 1 if you have the <ncurses.h> header file. */
/* #undef HAVE_NCURSES_H */

/* Define to 1 if you have the <netdb.h> header file. */
#define HAVE_NETDB_H 1

/* Define to 1 if netloc library found */
/* #undef HAVE_NETLOC */

/* Define to 1 if netloc includes use underscore not subdirectory */
/* #undef HAVE_NETLOC_NOSUB */

/* define if you have nrt.h */
/* #undef HAVE_NRT_H */

/* define if numa library installed */
/* #undef HAVE_NUMA */

/* Define to 1 if ofed library found */
/* #undef HAVE_OFED */

/* Define to 1 if using code with pma_query_via */
/* #undef HAVE_OFED_PMA_QUERY_VIA */

/* define if you have openssl. */
#define HAVE_OPENSSL 1

/* define if you have the PAM library */
/* #undef HAVE_PAM */

/* Define to 1 if you have the <pam/pam_appl.h> header file. */
/* #undef HAVE_PAM_PAM_APPL_H */

/* Define to 1 if you have the <paths.h> header file. */
#define HAVE_PATHS_H 1

/* define if you have permapi_h */
/* #undef HAVE_PERMAPI_H */

/* Define to 1 if pmix library found */
#define HAVE_PMIX 1

/* Define if you have Posix semaphores. */
/* #undef HAVE_POSIX_SEMS */

/* Define if libc sets program_invocation_name */
#define HAVE_PROGRAM_INVOCATION_NAME 1

/* Define if you have POSIX threads libraries and header files. */
#define HAVE_PTHREAD 1

/* Define to 1 if you have the <pthread.h> header file. */
#define HAVE_PTHREAD_H 1

/* Have PTHREAD_PRIO_INHERIT. */
#define HAVE_PTHREAD_PRIO_INHERIT 1

/* Define to 1 if you have the `ptrace64' function. */
/* #undef HAVE_PTRACE64 */

/* Define to 1 if you have the <pty.h> header file. */
#define HAVE_PTY_H 1

/* Define if you are compiling with readline. */
/* #undef HAVE_READLINE */

/* Define to 1 for running on a real Cray system */
/* #undef HAVE_REAL_CRAY */

/* Define to 1 if you have the `sched_setaffinity' function. */
#define HAVE_SCHED_SETAFFINITY 1

/* Define to 1 if you have the <security/pam_appl.h> header file. */
/* #undef HAVE_SECURITY_PAM_APPL_H */

/* Define to 1 if you have the `setproctitle' function. */
/* #undef HAVE_SETPROCTITLE */

/* Define to 1 if you have the `setresuid' function. */
#define HAVE_SETRESUID 1

/* Define to 1 if you have the <socket.h> header file. */
/* #undef HAVE_SOCKET_H */

/* Define to 1 if you have the `statfs' function. */
#define HAVE_STATFS 1

/* Define to 1 if you have the `statvfs' function. */
#define HAVE_STATVFS 1

/* Define to 1 if you have the <stdbool.h> header file. */
#define HAVE_STDBOOL_H 1

/* Define to 1 if you have the <stdint.h> header file. */
#define HAVE_STDINT_H 1

/* Define to 1 if you have the <stdlib.h> header file. */
#define HAVE_STDLIB_H 1

/* Define to 1 if you have the `strerror' function. */
#define HAVE_STRERROR 1

/* Define to 1 if you have the `strerror_r' function. */
#define HAVE_STRERROR_R 1

/* Define to 1 if you have the <strings.h> header file. */
#define HAVE_STRINGS_H 1

/* Define to 1 if you have the <string.h> header file. */
#define HAVE_STRING_H 1

/* Define to 1 if you have the `strlcpy' function. */
/* #undef HAVE_STRLCPY */

/* Define to 1 if you have the `strndup' function. */
#define HAVE_STRNDUP 1

/* Define to 1 if you have the `strsignal' function. */
#define HAVE_STRSIGNAL 1

/* Define to 1 if you have the `sysctlbyname' function. */
/* #undef HAVE_SYSCTLBYNAME */

/* Define to 1 if you have the <sysint.h> header file. */
/* #undef HAVE_SYSINT_H */

/* Define to 1 if you have the <sys/dr.h> header file. */
/* #undef HAVE_SYS_DR_H */

/* Define to 1 if you have the <sys/ipc.h> header file. */
#define HAVE_SYS_IPC_H 1

/* Define to 1 if you have the <sys/prctl.h> header file. */
#define HAVE_SYS_PRCTL_H 1

/* Define to 1 if you have the <sys/ptrace.h> header file. */
#define HAVE_SYS_PTRACE_H 1

/* Define to 1 if you have the <sys/sem.h> header file. */
#define HAVE_SYS_SEM_H 1

/* Define to 1 if you have the <sys/shm.h> header file. */
#define HAVE_SYS_SHM_H 1

/* Define to 1 if you have the <sys/socket.h> header file. */
#define HAVE_SYS_SOCKET_H 1

/* Define to 1 if you have the <sys/statfs.h> header file. */
#define HAVE_SYS_STATFS_H 1

/* Define to 1 if you have the <sys/statvfs.h> header file. */
#define HAVE_SYS_STATVFS_H 1

/* Define to 1 if you have the <sys/stat.h> header file. */
#define HAVE_SYS_STAT_H 1

/* Define to 1 if you have the <sys/sysctl.h> header file. */
#define HAVE_SYS_SYSCTL_H 1

/* Define to 1 if you have the <sys/syslog.h> header file. */
#define HAVE_SYS_SYSLOG_H 1

/* Define to 1 if you have the <sys/systemcfg.h> header file. */
/* #undef HAVE_SYS_SYSTEMCFG_H */

/* Define to 1 if you have the <sys/types.h> header file. */
#define HAVE_SYS_TYPES_H 1

/* Define to 1 if you have the <sys/vfs.h> header file. */
#define HAVE_SYS_VFS_H 1

/* Define to 1 if you have <sys/wait.h> that is POSIX.1 compatible. */
#define HAVE_SYS_WAIT_H 1

/* Define to 1 if you have the <termcap.h> header file. */
/* #undef HAVE_TERMCAP_H */

/* Define to 1 if you have the <unistd.h> header file. */
#define HAVE_UNISTD_H 1

/* Define to 1 if you have the <utmp.h> header file. */
#define HAVE_UTMP_H 1

/* Define to 1 if you have the <values.h> header file. */
#define HAVE_VALUES_H 1

/* Define if you have __progname. */
#define HAVE__PROGNAME 1

/* Define to 1 if you have the external variable, _system_configuration with a
   member named physmem. */
/* #undef HAVE__SYSTEM_CONFIGURATION */

/* Define to 1 if the system has the `__builtin_clzll' built-in function */
#define HAVE___BUILTIN_CLZLL 1

/* Define to 1 if the system has the `__builtin_ctzll' built-in function */
#define HAVE___BUILTIN_CTZLL 1

/* Define to 1 if the system has the `__builtin_popcountll' built-in function
   */
#define HAVE___BUILTIN_POPCOUNTLL 1

/* Defined if libcurl supports AsynchDNS */
/* #undef LIBCURL_FEATURE_ASYNCHDNS */

/* Defined if libcurl supports IDN */
/* #undef LIBCURL_FEATURE_IDN */

/* Defined if libcurl supports IPv6 */
/* #undef LIBCURL_FEATURE_IPV6 */

/* Defined if libcurl supports KRB4 */
/* #undef LIBCURL_FEATURE_KRB4 */

/* Defined if libcurl supports libz */
/* #undef LIBCURL_FEATURE_LIBZ */

/* Defined if libcurl supports NTLM */
/* #undef LIBCURL_FEATURE_NTLM */

/* Defined if libcurl supports SSL */
/* #undef LIBCURL_FEATURE_SSL */

/* Defined if libcurl supports SSPI */
/* #undef LIBCURL_FEATURE_SSPI */

/* Defined if libcurl supports DICT */
/* #undef LIBCURL_PROTOCOL_DICT */

/* Defined if libcurl supports FILE */
/* #undef LIBCURL_PROTOCOL_FILE */

/* Defined if libcurl supports FTP */
/* #undef LIBCURL_PROTOCOL_FTP */

/* Defined if libcurl supports FTPS */
/* #undef LIBCURL_PROTOCOL_FTPS */

/* Defined if libcurl supports HTTP */
/* #undef LIBCURL_PROTOCOL_HTTP */

/* Defined if libcurl supports HTTPS */
/* #undef LIBCURL_PROTOCOL_HTTPS */

/* Defined if libcurl supports IMAP */
/* #undef LIBCURL_PROTOCOL_IMAP */

/* Defined if libcurl supports LDAP */
/* #undef LIBCURL_PROTOCOL_LDAP */

/* Defined if libcurl supports POP3 */
/* #undef LIBCURL_PROTOCOL_POP3 */

/* Defined if libcurl supports RTSP */
/* #undef LIBCURL_PROTOCOL_RTSP */

/* Defined if libcurl supports SMTP */
/* #undef LIBCURL_PROTOCOL_SMTP */

/* Defined if libcurl supports TELNET */
/* #undef LIBCURL_PROTOCOL_TELNET */

/* Defined if libcurl supports TFTP */
/* #undef LIBCURL_PROTOCOL_TFTP */

/* Define the libnrt.so location */
/* #undef LIBNRT_SO */

/* Define to 1 for --get-user-env to load user environment without .login */
/* #undef LOAD_ENV_NO_LOGIN */

/* Define to the sub-directory where libtool stores uninstalled libraries. */
#define LT_OBJDIR ".libs/"

/* Define to 1 for memory leak debugging. */
/* #undef MEMORY_LEAK_DEBUG */

/* Enable multiple slurmd on one node */
/* #undef MULTIPLE_SLURMD */

/* Define to 1 if with non thread-safe code */
/* #undef MYSQL_NOT_THREAD_SAFE */

/* Define to 1 if you are building a production release. */
#define NDEBUG 1

/* Define to the address where bug reports for this package should be sent. */
#define PACKAGE_BUGREPORT "slurm-dev@schedmd.com"

/* Define to the full name of this package. */
#define PACKAGE_NAME "slurm"

/* Define to the full name and version of this package. */
#define PACKAGE_STRING "slurm 17.02"

/* Define to the one symbol short name of this package. */
#define PACKAGE_TARNAME "slurm"

/* Define to the home page for this package. */
#define PACKAGE_URL "https://slurm.schedmd.com"

/* Define to the version of this package. */
#define PACKAGE_VERSION "17.02"

/* Define PAM module installation directory. */
#define PAM_DIR ""

/* Define the project's name. */
#define PROJECT "slurm"

/* Define to necessary symbol if this constant uses a non-standard name on
   your system. */
/* #undef PTHREAD_CREATE_JOINABLE */

/* Define to 1 if ptrace takes five arguments. */
/* #undef PTRACE_FIVE_ARGS */

/* Define the project's release. */
#define RELEASE "1"

/* Define to 1 for salloc to kill child processes at job termination */
/* #undef SALLOC_KILL_CMD */

/* Define to 1 to require salloc execution in the foreground. */
/* #undef SALLOC_RUN_FOREGROUND */

/* Define to 1 if sched_getaffinity takes three arguments. */
#define SCHED_GETAFFINITY_THREE_ARGS 1

/* Define to 1 if sched_getaffinity takes two arguments. */
/* #undef SCHED_GETAFFINITY_TWO_ARGS */

/* Define if you need setproctitle padding */
#define SETPROCTITLE_PS_PADDING '\0'

/* Define to the setproctitle() emulation type */
#define SETPROCTITLE_STRATEGY PS_USE_CLOBBER_ARGV

/* Define path to sleep command */
#define SLEEP_CMD "/bin/sleep"

/* Define the default port number for slurmctld */
#define SLURMCTLD_PORT 6817

/* Define the default port count for slurmctld */
#define SLURMCTLD_PORT_COUNT 1

/* Define the default port number for slurmdbd */
#define SLURMDBD_PORT 6819

/* Define the default port number for slurmd */
#define SLURMD_PORT 6818

/* API current age */
#define SLURM_API_AGE 0

/* API current version */
#define SLURM_API_CURRENT 31

/* API current major */
#define SLURM_API_MAJOR 31

/* API current rev */
#define SLURM_API_REVISION 0

/* Define the API's version */
#define SLURM_API_VERSION 0x1f0000

/* Define if your architecture's byteorder is big endian. */
/* #undef SLURM_BIGENDIAN */

/* Define the project's major version. */
#define SLURM_MAJOR "17"

/* Define the project's micro version. */
#define SLURM_MICRO "2"

/* Define the project's minor version. */
#define SLURM_MINOR "02"

/* Define Slurm installation prefix */
#define SLURM_PREFIX "/home/slurm/slurm_sim_ws/slurm_opt"

/* Define SLURM_SIMULATOR if you are building slurm in simulator mode. */
#define SLURM_SIMULATOR 1

/* SLURM Version Number */
#define SLURM_VERSION_NUMBER 0x110202

/* Define the project's version string. */
#define SLURM_VERSION_STRING "17.02.2"

/* Define to 1 if you have the ANSI C header files. */
#define STDC_HEADERS 1

/* Define to 1 if strerror_r returns char *. */
/* #undef STRERROR_R_CHAR_P */

/* Define path to su command */
#define SUCMD "/bin/su"

/* 3-dimensional architecture */
/* #undef SYSTEM_DIMENSIONS */

/* Define to 1 if you can safely include both <sys/time.h> and <time.h>. */
#define TIME_WITH_SYS_TIME 1

/* Define slurm_ prefix function aliases for plugins */
#define USE_ALIAS 1

/* define if using ISO 8601 time format */
#define USE_ISO_8601 /**/

/* Define the project's version. */
#define VERSION "17.02.2"

/* Define WORDS_BIGENDIAN to 1 if your processor stores words with the most
   significant byte first (like Motorola and SPARC, unlike Intel). */
#if defined AC_APPLE_UNIVERSAL_BUILD
# if defined __BIG_ENDIAN__
#  define WORDS_BIGENDIAN 1
# endif
#else
# ifndef WORDS_BIGENDIAN
/* #  undef WORDS_BIGENDIAN */
# endif
#endif

/* Define curl_free() as free() if our version of curl lacks curl_free. */
/* #undef curl_free */

/* Define to rpl_malloc if the replacement function should be used. */
/* #undef malloc */
